### PR TITLE
APPCAFE-03: When selecting a row it shows CollectionViewController with mocked data

### PR DESCRIPTION
**Pull request by 1699kratos**
## ISSUE: APPCAFE-03: 
https://trello.com/c/PACum6Dn

## FIX: 
when user tap a row it appears a collection view controller with the products of that cafe, again mocked data and the design is the one you showed me last time

## EVIDENCES: 
image

### CODE REVIEW

File: Prices/Prices/MenuViewController.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    
+    let items = ["Doritos Nacho", "Tostitos Salsa Verde", "Doritos Nacho", "Tostitos Salsa Verde"]
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return  items.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let item = collectionView.dequeueReusableCell(withReuseIdentifier: "itemCell", for: indexPath) as! CollectionViewCell
+       
+
```
Comment (circesofia) [inline Prices/Prices/MenuViewController.swift:20]: Please delete empty lines

File: Prices/Prices/MenuViewController.swift [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        let nibCell = UINib(nibName: "CollectionViewCell", bundle: nil)
+        menuCollection.register(nibCell, forCellWithReuseIdentifier: "itemCell")
+        menuCollection.dataSource = self
+        menuCollection.delegate = self
+        
+    }
+    
+    
+
+
```
Comment (circesofia) [inline Prices/Prices/MenuViewController.swift:39]: delete empty lines here too

File: Prices/Prices/MenuViewController.swift [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+class MenuViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate,UICollectionViewDelegateFlowLayout  {
+    
+    let items = ["Doritos Nacho", "Tostitos Salsa Verde", "Doritos Nacho", "Tostitos Salsa Verde"]
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return  items.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let item = collectionView.dequeueReusableCell(withReuseIdentifier: "itemCell", for: indexPath) as! CollectionViewCell
```
Comment (circesofia) [inline Prices/Prices/MenuViewController.swift:18]: remember to never force unwrap

File: Prices/Prices/CollectionViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+class CollectionViewCell: UICollectionViewCell {
+
+    @IBOutlet weak var itemPrice: UILabel!
+    @IBOutlet weak var itemName: UILabel!
+    @IBOutlet weak var itemImage: UIImageView!
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+    }
+
```
Comment (circesofia) [inline Prices/Prices/CollectionViewCell.swift:19]: delete empty lines

File: Prices/Prices/CafeteriasViewController.swift [hunk 5]
```diff
@@ -15,6 +15,7 @@ class CafeteriasViewController: UIViewController {
         tableView.delegate = self
         tableView.dataSource = self
     }
+
```
Comment (circesofia) [inline Prices/Prices/CafeteriasViewController.swift:18]: why is this necesary?

**Review by circesofia — CHANGES_REQUESTED** [review]
The PR format is all wrong
The name of the PR should be the name of the ticket in trello
The issue part should have the link to the trello ticket

File: Prices/Prices/CafeteriasViewController.swift (hunk 5 above)
Comment (hachiosaki) [inline Prices/Prices/CafeteriasViewController.swift:18]: was this necessary?

File: Prices/Prices/CollectionViewCell.swift (hunk 4 above)
Comment (hachiosaki) [inline Prices/Prices/CollectionViewCell.swift:19]: delete empty lines

File: Prices/Prices/MenuViewController.swift (hunk 2 above)
Comment (hachiosaki) [inline Prices/Prices/MenuViewController.swift:39]: delete empty lines

File: Prices/Prices/MenuViewController.swift (hunk 3 above)
Comment (hachiosaki) [inline Prices/Prices/MenuViewController.swift:18]: never force unwrap

File: Prices/Prices/MenuViewController.swift (hunk 1 above)
Comment (hachiosaki) [inline Prices/Prices/MenuViewController.swift:20]: delete empty lines

**Review by hachiosaki — CHANGES_REQUESTED** [review]
The PR format is all wrong
The name of the PR should be the name of the ticket in trello
The issue part should have the link to the trello ticket
Please add images of the products, not only Tostitos verdes
VERY IMPORTANT: you cannot merge branches there's an error in APPCAFE-02 that now is here too: I'm still seeing this DS_Store file
Captura de Pantalla 2021-10-15 a la(s) 20 39 03

File: Prices/Prices/CafeteriasViewController.swift (hunk 5 above)
Comment (1699kratos) [reply Prices/Prices/CafeteriasViewController.swift:18]: sorry

**Pull request merged**